### PR TITLE
Add page on shader chunk migrations

### PR DESCRIPTION
**Pull request by slimbuck**
Screenshot 2022-07-20 at 12 56 27

### CODE REVIEW

File: content/en/user-manual/graphics/shader-chunk-migrations.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+| `lightmapAddPS`, `lightmapDirAddPS` | <ul><li>new chunks for adding the lightmap values passed in from the backend</li></ul> |
+| `lightSpecularAnisoGGXPS` | <ul><li>added clear coat `#define`</li></ul> |
+| `lightSpecularBlinnPS`, `lightSpecularPhongPS` | <ul><li>added clear coat `#define`, removed call to `antiAliasGlossiness()`</li></ul> |
+| `normalVertexPS` | <ul><li>removed chunk, moved functionality to `normalMapPS` frontend chunk</li></ul> |
+| `specularAaNonePS`, `specularAaToksvigPS`, `specularAaToksvigFastPS` | <ul><li>removed</li></ul> |
+| `startPS` | <ul><li>removed global declarations, generate them on demand instead</li></ul> |
+
+
+[1]: https://github.com/playcanvas/engine/issues/4250
+[2]: https://github.com/playcanvas/engine/releases/tag/v1.55.0
```
Comment (mvaligursky) [inline content/en/user-manual/graphics/shader-chunk-migrations.md:53]: should something reference this [2] link?

**Comment by mvaligursky** [comment]
Could you please post a screenshot of the generated page.

File: content/en/user-manual/graphics/shader-chunk-migrations.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+title: Shader Chunk Migrations
+layout: usermanual-page.hbs
+position: 6
+---
+
+## Introduction
+
+The PlayCanvas Engine's material shader chunk system is undergoing substantial changes in order to support a more flexible material system. Please see [this page][1] for more context.
+
+In order to help users migrate their existing custom shader chunks, this page lists the changes made to chunks and organizes them by engine release (starting v1.51).
```
Comment (yaustar) [inline content/en/user-manual/graphics/shader-chunk-migrations.md:11]: Should this be v1.55?

File: content/en/user-manual/graphics/shader-chunk-migrations.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+## Chunk API Versions
+
+The debug version of the Engine will report any API changes to the console when it detects overridden chunks at runtime.
+
+Once an application's chunks have been updated to the latest API they must be flagged as such. For example, after updating a material's custom chunks to the latest engine release (say v1.55), specify this in the chunks object as follows:
+```javascript
+material.chunks.diffusePS = '...';
+material.chunks.APIVersion = pc.CHUNKAPI_1_55;
+```
+
```
[the remark this answers is not in the document]
Comment (slimbuck) [reply content/en/user-manual/graphics/shader-chunk-migrations.md:24]: It does the opposite, I've added a sentence.

**Review by yaustar — APPROVED** [review]

**Pull request merged**